arch/arm/boot/dts/stm32mp15xx-dhcom-som.dtsi

```diff
@@ -421,6 +426,10 @@
 	status = "okay";
 };
 
+&optee {
```

Comment:
What does this node mean?

Reply:
This node enables the OP-TEE node added in https://github.com/linaro-swg/linux/commit/1aadb75d20cdc69041cb0c0de5953397c241ccf9.

Reply:
OK, thanks.